src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/integration/endpoint/ExternalDocRequestEndpointNoSNSIntTest.kt

```diff
@@ -1,56 +0,0 @@
-package uk.gov.justice.digital.hmpps.crimeportalgateway.integration.endpoint
```

Comment:
This is a bit too hard to test now - the SNS topic is always going be there as it is created automatically by the hmpps-spring-boot-sqs library